src/xpi.js

```diff
@@ -85,7 +85,7 @@ export default class Xpi {
             resolve(readStream);
           });
         })
-        .catch(err => reject(err));
+        .catch((err) => reject(err));
```

Comment:
I find using the braces unconditionally in arrow functions makes the code easier to scan and consistent. If anyone is really against this though, I can back it out.